### DISCUSSION TITLE
Why the command openssl displays its output through stderr?

### DISCUSSION THREAD

**Discussion by bangcheng** [openssl/openssl#21600] [Q&A] [answered]
When I use the `openssl help` command, I found that the help information is output to `stderr`. This is not the same as other software, I am confused. Is there any reason for this way?

**Comment by paulidale** [comment, marked as answer]
My guess would be that this is how it was originally done and that it has stayed that way since.

A lot of _modern_ command line tool conventions didn't exist when the project began.
